extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -106,7 +120,13 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
         .getVisibilityState();
     const adTestOn = this.element.getAttribute('data-adtest') ||
         isInManualExperiment(this.element);
-    const format = `${slotRect.width}x${slotRect.height}`;
+    const width = slotRect.width;
+    // If responsive, set height to value that is considered valid by
+    // ad server.
+    const height =
```

Comment:
Is this a blocking item for this PR?  I'd prefer to get it in ASAP and we can always optimize later (other option would be to override onLayoutMeasure and do it there although getAdUrl _should_ execute shortly thereafter).